src/service/triple-store-service.js

```diff
@@ -26,462 +32,216 @@ class TripleStoreService {
         }
     }
 
-    async localStoreAsset(
+    async insertKnowledgeCollection(
         repository,
-        assertionId,
-        assertion,
-        blockchain,
-        contract,
-        tokenId,
-        keyword,
+        knowledgeCollectionUAL,
+        knowledgeAssetsUALs,
+        triples,
```

Comment:
should this be called triples? on the client we allow user to create by passing triples/quads so maybe calling it quads is better also to be consistent. or should we allow triples only on the client side?

Reply:
This should be `statements` perhaps, doesn't matter which format it is